.storybook-s2/docs/Release Notes.mdx

```diff
@@ -4,6 +4,37 @@ export default MDXLayout;
 
 # Release Notes
 
+## v0.6.0
+
+We've...
+
+### New Components
+
+* feat: Add S2 ActionBar component - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/7446)
+
+### Updates
+
+* [Button](?path=/docs/button--docs) - feat: add gradient variants to Button - [@jluyau](https://github.com/jluyau) - [PR](https://github.com/adobe/react-spectrum/pull/7514)
```

Comment:
Can you clean up all these linting bits? They were meant to be used to categorize the commits, not actually appear in the release notes.
All the "feat:" "[component link]" etc.
Look at the last release to copy the formatting.

Reply:
also the S2 release notes usually have a different format than v3, e.g. not including every commit.

Reply:
Yes. I should have made this a WIP.